frontend/ui/data/onetime_migration.lua

```diff
@@ -331,7 +331,7 @@ if last_migration_date < 20220116 then
         for i = #opds_servers, 1, -1 do
             local server = opds_servers[i]
 
-            if server.url == "https://standardebooks.org/opds" then
+            if server.url == "https://standardebooks.org/feeds/opds" then
```

Comment:
This line shouldn't be changed. Quite the contrary. ;-)

@NiLuJe did you write these lines originally? What do you suppose is the best way to apply them again but with 20220725 instead, just changing it a few lines up?

Reply:
It needs a new block altogether, with a different revision number ;).